android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -69,6 +69,11 @@ private Boolean isTablet() {
     return layout == Configuration.SCREENLAYOUT_SIZE_LARGE || layout == Configuration.SCREENLAYOUT_SIZE_XLARGE;
   }
 
+  @ReactMethod
+  public void getDeviceInfo(@NonNull final Promise promise) {
+    promise.resolve(this.getConstants().get("DeviceInfo"));
+  }
+
   @Override
   public @Nullable Map<String, Object> getConstants() {
```

Comment:
I'm proposing we move over to using the getter 100% of the time instead of the JS app using the constants sometimes and the getter other times.

My real question is: 
It seems like we're adding a getter method because some of the device info properties might change (like the timezone or 24 hour time). If these properties might change, we shouldn't be including them in the `constants` if they aren't really constants.